go.mod

```diff
@@ -20,12 +20,13 @@ require (
 	// This PR: https://github.com/helm/helm/pull/11659 fixes the issue
 	// and has not been merged to main. Adding this temporarily to fix this issue
 	helm.sh/helm/v3 v3.9.0-rc.1.0.20221222223505-5c40996dad38
-	k8s.io/api v0.25.2
-	k8s.io/apimachinery v0.25.2
-	k8s.io/client-go v0.25.2
+	k8s.io/api v0.25.3
+	k8s.io/apimachinery v0.25.3
+	k8s.io/client-go v0.25.3
 	oras.land/oras-go/v2 v2.0.0-rc.6
-	sigs.k8s.io/controller-runtime v0.11.1
+	sigs.k8s.io/controller-runtime v0.13.0
```

Comment:
This is a significant version bump. Will this lead to any compatibility issue with existing K8 versions we support?

Reply:
Works locally so far, but guess we can test things out a bit in dev with this.